build/common.props

```diff
@@ -20,4 +20,7 @@
     <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(RoslynVersion)" PrivateAssets="All" />
```

Comment:
This should be doable for now, but become hard to do eventually. 

Right now the only difference is that the new compiler has extra checks. However at some point you will need to use 7.2 features such as `ref struct` . Then the same source will not be buildable by both compilers unless you use `#if`